app/views/components/view_application.scala.html

```diff
@@ -0,0 +1,269 @@
+                <div>
+                    <dt class="cya-question">@messages("areYouSendingSamples.checkYourAnswersLabel")</dt>
+                    <dd class="cya-answer">@{if(pdf.sendingSample) {messages("site.yes")} else {messages("site.no")}}</dd>
+                </div>
+                @if(pdf.hazardousSample) {
+                <div>
+                    <dt class="cya-question">@messages("isSampleHazardous.checkYourAnswersLabel")</dt>
+                    <dd class="cya-answer">@{if(pdf.hazardousSample.get) {messages("site.yes")} else {messages("site.no")}}</dd>
+                </div>
+
+                <div>
+                    <dt class="cya-question">@messages("returnSamples.checkYourAnswersLabel")</dt>
+                    <dd class="cya-answer">@{if(pdf.returnSample) {messages("site.yes")} else {messages("site.no")}}</dd>
+                </div>
+                }
+                } else {
+                <div>
+                    <dt class="cya-question">@messages("areYouSendingSamples.checkYourAnswersLabel")</dt>
+                    <dd class="cya-answer">@messages("site.no")</dd>
+                </div>
+                }
+
+                @if(pdf.foundCommodityCode.isDefined) {
+                <div>
+                    <dt class="cya-question">@messages("application.section.aboutItem.envisagedCommodityCode")</dt>
+                    <dd class="cya-answer">@pdf.foundCommodityCode</dd>
+                </div>
+                } else {
+                <div>
+                    <dt class="cya-question">@messages("view.applicationPdf.foundComodityCode")</dt>
+                    <dd class="cya-answer">@messages("site.no")</dd>
+                </div>
+                }
+
+                @if(pdf.legalProblems.isDefined) {
+                <div>
+                    <dt class="cya-question">@messages("view.applicationPdf.legalProblem")</dt>
+                    <dd class="cya-answer">@pdf.legalProblems</dd>
+                </div>
+                } else {
+                <div>
+                    <dt class="cya-question">@messages("view.applicationPdf.legalProblemQuestion")</dt>
+                    <dd class="cya-answer">@messages("site.no")</dd>
+                </div>
+                }
+
+            </dl>
+            <h3 class="heading-medium">@messages("beforeYouStart.subheading2")</h3>
+            <dl class="govuk-check-your-answers cya-questions-short">
+                <div>
+                    <dt class="cya-question">@messages("previousBTIRuling.checkYourAnswersLabel")</dt>
+                    <dd class="cya-answer">@{if(pdf.returnSample) {messages("site.yes")} else {messages("site.no")}}</dd>
+                </div>
+            </dl>
+            <div class="footer-info footer-page-break">
+                <span class="footer-page-no"></span>
+                <span class="footer-page-date">HMRC @Dates.formatForPdf(pdf.dateSubmitted)</span>
+           </div>
+        </div>
+    </div>
+</div>
+</div>
```

Comment:
I don't think it's urgent but it might be nice to break this view up into some components

Reply:
Will do this in separate ticket may be